spec/controllers/user/registration_controller_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe User::RegistrationsController, type: :controller do
+  before do
+    @request.env["devise.mapping"] = Devise.mappings[:user]
+
+    allow(APP_CONFIG).to receive(:dig).with(:hcaptcha, :enabled).and_return(true)
+    allow(controller).to receive(:verify_hcaptcha).and_return(captcha_successful)
+  end
+
+  describe "#create" do
+    context "valid user sign up" do
+      let :registration_params do
+        {
+            user: {
+                screen_name: 'dio',
+                email: 'the-world-21@somewhere.everywhere',
+                password: 'AReallySecurePassword456!',
+                password_confirmation: 'AReallySecurePassword456!'
+            }
+        }
+      end
+
+      subject { post :create, params: registration_params }
+
+      context "when captcha was invalid" do
+        let(:captcha_successful) { false }
+        it "doesn't allow a registration without an invalid captcha" do
```

Comment:
`without an invalid captcha` is not only wrong here, but also made redundant by the `context` description